src/main/resources/config/application.yml

```diff
@@ -119,6 +119,14 @@ artemis:
             short-name: "artemis-build-agent-1"
             display-name: "Artemis Build Agent 1"
 
+    compatible-versions:
```

Comment:
should we really force the users directly to the latest version?

Reply:
In today's meeting, it was decided to use the latest versions